### PR TITLE
Users should be able to click anywhere on the row to drill down on a service

### PR DESCRIPTION
**Pull request by chalettu**
PV https://www.pivotaltracker.com/story/show/144791027

### CODE REVIEW

**Comment by chalettu** [comment]
@miq-bot add_label bug
@miq-bot add_label fine/yes
@miq-bot add_label services

**Comment by chalettu** [comment]
@miq-bot add_label my services

File: client/assets/sass/_explorer.sass [hunk 1]
```diff
@@ -62,3 +62,6 @@
 .card-view-pf
   .card
     cursor: pointer
+
+.service-explorer-row
```
Comment (AllenBW) [inline client/assets/sass/_explorer.sass]: Lets try to keep the names explorer agnostic, rather service-explorer, prefer `explorer-row`

**Comment by simaishi** [comment]
@chalettu @chriskacerguis Is there a BZ associated with this change?  Need a BZ to backport to Fine.

**Comment by chriskacerguis** [comment]
@simaishi there was no BZ.

**Comment by simaishi** [comment]
@chalettu BZ is required to backport to Fine branch. If this is still needed in Fine, please add a BZ link.

**Pull request merged**